clients/src/test/java/org/apache/kafka/common/record/DefaultRecordBatchTest.java

```diff
@@ -63,15 +61,7 @@
 import static org.mockito.Mockito.when;
 
 public class DefaultRecordBatchTest {
-    private static final Random RANDOM;
-
-    static {
-        try {
-            RANDOM = SecureRandom.getInstanceStrong();
-        } catch (NoSuchAlgorithmException e) {
-            throw new RuntimeException(e);
-        }
-    }
+    private static final Random RANDOM = new Random();
```

Comment:
Perhaps a comment would be useful here to point out why it is preferable to avoid `SecureRandom.getInstanceStrong()` in tests.